DEPRECATED.md

```diff
@@ -27,7 +27,7 @@ The extension won't receive any updates, bug fixes, or security fixes, so you us
 
 ## Is there any action for me?
 
-If you want to keep using the extension, no, there's no action.
```

Comment:
I think this line is correct, since if you're using the extension today and want to keep using it, you don't have to do anything, it'll stay installed, even after sunset. 
But it should have two spaces at the end, for line break.